app/queries/subscribers_for_immediate_email_query.rb

```diff
@@ -4,20 +4,7 @@ def self.call
   end
 
   def call
-    Subscriber
-      .activated
-      .where(
-        unprocessed_subscription_contents_exist_for_subscribers
-       )
-  end
-
-private
-
-  def unprocessed_subscription_contents_exist_for_subscribers
-    SubscriptionContent
-      .joins(:subscription)
-      .where(email_id: nil)
-      .where("subscriptions.subscriber_id = subscribers.id")
-      .exists
+    subscriber_ids = Subscription.joins(:subscription_contents).where(subscription_contents: { email_id: nil }).select(:subscriber_id)
```

Comment:
It would be good to format this like it was, with the different parts of the query on their own lines, avoid having one really one line.